independent-projects/tools/common/src/test/java/io/quarkus/cli/commands/ListExtensionsTest.java

```diff
@@ -36,7 +36,7 @@ public void listWithBom() throws IOException {
 
         FileProjectWriter writer = new FileProjectWriter(pom.getParentFile());
         new CreateProject(writer)
-                .groupId(getPluginGroupId())
+                .groupId("org.acme")
```

Comment:
why are all these "org.acme" reference replaced? shouldn't they stay the same for this test ?

Reply:
Because the groupId of quarkus-maven-plugin is not related to the groupId of a random test project. Or am I missing something?

Reply:
ooooh - I misread the diff. you are right. your fix is correct.